### PR TITLE
Update normalize

### PR DESCRIPTION
**Pull request by ericmarc159**

### CODE REVIEW

File: src/newNormalize.js [hunk 1]
```diff
@@ -2,57 +2,96 @@
...
+      deleteFlag
+    );
+    result['ROOT_MUTATION'] = mutationQuery.output;
+
+    //iterate thru the different response objects that were mutated
+    mutationQuery.names.forEach((name) => {
+      const obj = resultObj.data[name];
+      //checks if the current element is an array
+      if (Array.isArray(obj)) {
+        //iterates thru the array of objects and stores the hash in the result object with 'DELETE' as value
```
Comment (dsymiller) [inline src/newNormalize.js:58]: all 'DELETE' should be 'DELETED'

File: src/newNormalize.js [hunk 2]
```diff
@@ -114,3 +153,264 @@ function createHash(obj, output = {}) {
...
+        __typename: 'meta',
+        id: 'scalar',
+      },
+    },
+  ],
+};
+
+const resultObject1 = {
+  data: {
+    deleteMovie: [
```
Comment (dsymiller) [inline src/newNormalize.js:230]: to match the above example this shouldn't be an array

File: src/newNormalize.js [hunk 3]
```diff
@@ -114,3 +153,264 @@ function createHash(obj, output = {}) {
...
+    ],
+  },
+};
+
+const resultObj1 = {
+  ROOT_QUERY: {
+    'movies(input:{genre:ACTION})': ['Movie~1', 'Movie~4'],
+    actors: ['Actor~1', 'Actor~2', 'Actor~3', 'Actor~4', 'Actor~5'],
+  },
+  ROOT_MUTATION: { 'deleteMovie(id:4)': ['Movie~4'] },
```
Comment (dsymiller) [inline src/newNormalize.js:368]: to match above example this shouldn't be an Array

File: src/newNormalize.js [hunk 4]
```diff
@@ -2,57 +2,96 @@
...
+              }
+            }
           }
         }
       }
     }
   }
+
+  // checks if there is a delete mutation
+  if (deleteFlag) {
```
Comment (dsymiller) [inline src/newNormalize.js:44]: How does normalize handle mutations when the delete flag is not set to true? I'm not seeing functionality for that.

**Review by dsymiller — CHANGES_REQUESTED** [review]

File: src/newNormalize.js [hunk 5]
```diff
@@ -2,57 +2,96 @@
...
+  }
+
   return result;
 }
 
 // creates the hashes for query requests and stores the reference has that will be stored in result
-function createRootQuery(queryObj, resultObj) {
+function createRootQuery(queryObj, resultObj, deleteFlag) {
   const output = {};
-  const query = queryObj.queries;
```
Comment (dsymiller) [inline src/newNormalize.js:43]: do we still need this new name funcitonality?

**Review by dsymiller — APPROVED** [review]
Looks good!

**Review by EsmaShr — APPROVED** [review]

**Pull request merged**